actors/builtin/miner/miner_actor.go

```diff
@@ -457,7 +457,9 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 			rt.Abortf(exitcode.ErrIllegalState, "sector %v already committed", params.SectorNumber)
 		}
 
-		validateExpiration(rt, rt.CurrEpoch(), params.Expiration, params.SealProof)
+		// Require sector lifetime meets minimum by assuming activation happens at last epoch permitted for seal proof.
+		maxActivation := rt.CurrEpoch() + MaxSealDuration[params.SealProof]
```

Comment:
So, this will allow the sector to potentially exceed the maximum lifetime (if it activates early). Is that OK? If so, we should document it somewhere.

Reply:
I'm fairly confident this is the calculation that @nicola verbally expressed to me. Yes, an extra comment sentence here would be good.

Reply:
If I am not wrong, I will check this with @irenegia, the alternative is having some PreCommits that pass, but their ProveCommit fails (if we check expiration at ProveCommit).